File: Basic-Video-Background-Blur-Java/app/src/main/java/com/tokbox/sample/basicvideochat/network/EmptyCallback.java
package com.tokbox.sample.basicvideochat.network;

import retrofit2.Call;
import retrofit2.Callback;
import retrofit2.Response;

public class EmptyCallback<T> implements Callback<T> {
    @Override
    public void onResponse(Call<T> call, Response<T> response) {

    }

    @Override
    public void onFailure(Call<T> call, Throwable t) {

    }
}

File: Basic-Video-Chat-Kotlin/app/src/main/java/com/tokbox/sample/basicvideochat/network/EmptyCallback.kt
package com.tokbox.sample.basicvideochat.network

import retrofit2.Call
import retrofit2.Callback
import retrofit2.Response

class EmptyCallback<T> : Callback<T> {

    override fun onResponse(call: Call<T>, response: Response<T>) {}
    override fun onFailure(call: Call<T>, t: Throwable) {}
}